lib/node_modules/@stdlib/math/base/special/factorial2f/test/fixtures/python/runner.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2025 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate fixtures."""
+
+import os
+import json
+import numpy as np
+from scipy.special import factorial2
+
+# Get the file path:
+FILE = os.path.realpath(__file__)
+
+# Extract the directory in which this file resides:
+DIR = os.path.dirname(FILE)
+
+
+def gen(x, name):
+    """Generate fixture data and write to file.
+
+    # Arguments
+
+    * `x`: domain
+    * `name::str`: output filename
+
+    # Examples
+
+    ``` python
+    python> x = linspace(-1000, 1000, 2001)
+    python> gen(x, './data.json')
+    ```
+    """
+    y = factorial2(x, exact=False)
```

Comment:
```suggestion
    y = factorial2(x, exact=False).astype(np.float32)
```

However, this change still doesn't affect the return values of the test fixtures.